TechnologyAssessmentApp/TechnologyAssessmentApp.xcodeproj/project.pbxproj

```diff
@@ -240,6 +252,7 @@
 		E54FA1DE20E23DF6002A4000 /* TechnologyAssessmentApp */ = {
 			isa = PBXGroup;
 			children = (
+				E578330A213076CE009DA6C1 /* POP Pattern */,
```

Comment:
Why pop pattern folder ? You should keep the same folder structure as you have. Do not create folder for activities, it's one same project for all activities